app/src/main/res/layout-sw600dp-port/revision_card_fragment.xml

```diff
@@ -0,0 +1,63 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.topic.revisioncard.RevisionCardViewModel" />
+  </data>
+
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/oppiaLightGreen"
+    android:overScrollMode="never"
+    android:scrollbars="none">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:gravity="center"
+      android:orientation="horizontal">
+
+      <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:minWidth="512dp"
+        android:orientation="vertical"
+        android:paddingBottom="160dp">
+
+        <TextView
+          android:id="@+id/revision_card_explanation_text"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="28dp"
+          android:layout_marginTop="36dp"
+          android:layout_marginEnd="28dp"
+          android:fontFamily="sans-serif"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="16sp" />
+
+        <Button
+          android:id="@+id/revision_card_return_button"
+          style="@style/StateButtonActive"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          android:layout_marginStart="28dp"
```

Comment:
Remove start and end margin as we have already mentioned minWidth.

Reply:
Done